DEVELOPER_GUIDE.md

```diff
@@ -2,74 +2,71 @@
+
+So the solution sometimes require some creativity like thinking new names that make it clear what the function does without departing too far from existing naming conventions.
 
-So the solution sometimes require some creativity like thinking new names that make it clear what the function does without departing too far from existing naming conventions. 
- 
 Another interesting case is the `zip` related functions:
 
- - For collections like types, in F# core it's normal to find `zip` / `map2` functions, which acts pairwise. But another possible implementation is the applicative zip which works cross product.
- - Here we define 2 generic functions, `lift2` which corresponds always to the applicative instance, so normally in non-collections it will correspond to the non-generic `map2` but since collections already have (or at least it's expected to) a `map2` acting pairwise, in those cases only we provide a `lift2` extension and that's what's used for its generic counterpart.
- 
- - The other generic function we provide is `zip` which is available mainly for collection like types and although they match the non-generic name, note that the behavior is not exactly the same, because F# core throws errors for list and arrays when the number of elements are different. So we could say that in fact it matches `.zipShortest` when defined, otherwise it matches `.zip`.
+- For collections like types, in F# core it's normal to find `zip` / `map2` functions, which acts pairwise. But another possible implementation is the applicative zip which works cross product.
+- Here we define 2 generic functions, `lift2` which corresponds always to the applicative instance, so normally in non-collections it will correspond to the non-generic `map2` but since collections already have (or at least it's expected to) a `map2` acting pairwise, in those cases only F#+ provides a `lift2` extension and that's what's used for its generic counterpart.
+- The other provided generic function is `zip` which is available mainly for collection like types and although they match the non-generic name, note that the behaviour is not exactly the same, because F# core throws errors for list and arrays when the number of elements are different. In that sense, in fact, it matches `.zipShortest` when defined, otherwise it matches `.zip`.
 
-So these zip related functions are a good example of how we do our best to fix FSharp.Core inconsistencies in a non-intrusive way.
+So these zip related functions are a good example of how F#+ does its best to fix FSharp.Core inconsistencies in a non-intrusive way.
 
 
 ### Abstractions
 
-The abstractions represented in the Control namespace of this library are a set of types that in fact represents generic functions, we call these types and we'll refer to as "Invokables".
+The abstractions represented in the Control namespace of this library are a set of types that in fact represents generic functions, these types are referred to as `Invokables`.
```

Comment:
IMO we shouldn't use code blocks for "Invokables", because there would be no use of "Invokable" in F# code while using F#+ (unlike `inline`).